app/models/journeys/further_education_payments/slug_sequence.rb

```diff
@@ -165,7 +183,10 @@ def slugs
         if poor_performance_form.completed_or_valid? && !answers.subject_to_problematic_actions?
           array << SLUGS_HASH["check-your-answers-part-one"]
           array << SLUGS_HASH["eligible"]
-          array << SLUGS_HASH["sign-in"]
+          if !answers.previously_claimed? && answers.have_one_login_account == "false" ||
+              !answers.previously_claimed? && answers.have_one_login_account == "i_dont_know"
```

Comment:
I'd be tempted to add a helper method to the answers
```
def has_one_login_account?
  have_one_login_account == "true"
end
``` 

so this could be `if !answers.previouusly_claimed? || !answers.has_one_login_account?`